styles.css

```diff
@@ -594,6 +592,119 @@ textarea[name=userMessage] {
   color: var(--sec-bgcolor);
 }
 
+/* Modal Styles */
+.modal {
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background: rgba(0, 0, 0, 0.5);
+  z-index: 3;
+}
+
+.modal-content {
+  border-radius: 16px;
+  width: 90vw;
+  height: 95vh;
+  background-color: #fff;
+}
+
+.modal-title-times {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 1rem;
+}
+
+.modal-title {
+  font-family: var(--pry-font-family);
+  font-weight: 800;
+  font-size: 1.5rem;
+  line-height: 2.75rem;
+  color: #172b4d;
+}
+
+.cls-btn {
+  cursor: pointer;
+  color: #67798e;
+  font-weight: 700;
+  font-size: 1.75rem;
+}
+
+.modal-year {
+  display: flex;
+  margin: 0 0 0 18px;
+}
+
+.modal-img {
```

Comment:
- [ ] it would be better if you adjust the image to be bigger lilke the one in the Figma design in order to match the design 

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/167958734-7113ba73-ce16-4167-b211-f205e5a10122.png)

- [ ] Yours 
![image](https://user-images.githubusercontent.com/61790587/167958755-fd47fd25-c787-470f-9786-dfd6c2df56f3.png)

Reply:
For the desktop version right?